src/python/seccomp.pyx

```diff
@@ -1044,5 +1046,28 @@ cdef class SyscallFilter:
         if rc != 0:
             raise RuntimeError(str.format("Library error (errno = {0})", rc))
 
+    def export_bpf_buf(self):
+        """ Export the filter in BPF format.
+
+        Description:
+        Return the filter in Berkeley Packet Filter (BPF) as bytes.
+        The output is identical to what is loaded into the Linux Kernel.
+        """
+        cdef size_t len = 0
+
+        # Figure out how big the program is.
+        rc = libseccomp.seccomp_export_bpf_buf(self._ctx, NULL, <size_t *>&len)
+        if rc != 0:
+            raise RuntimeError(str.format("Library error (errno = {0})", rc))
+
+        # Get the program.
+        cdef array.array data = array.array('b', bytes(len))
```

Comment:
Should this be `'b'` for a signed char or `'B'` for an unsigned char?

https://docs.python.org/3/library/array.html

Similarly, should the line below be `unsigned char[:]`?

Reply:
done